### PR TITLE
[Instrumentation.AspNetCore] Document metrics

### PR DESCRIPTION
**Pull request by Kielek**
Towards https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2292

## Changes

Changes based on https://github.com/open-telemetry/opentelemetry-dotnet/pull/4216/files

Document how to add AspNetCore metrics.
Document list of AspNetCore metrics.
Change Jeager exporter to Console exporter (as Jeager will be deprecated shortly). ~~Console exporter from Http example is not good idea for the ASP.NET Core application.~~

## Merge requirement checklist

* [x] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* ~~[ ] Unit tests added/updated~~
* ~~[ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes~~
* ~~[ ] Changes in public API reviewed (if applicable)~~

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.AspNetCore/README.md [hunk 1]
```diff
@@ -58,10 +58,43 @@ public void ConfigureServices(IServiceCollection services)
...
+    services.AddOpenTelemetry()
+        .WithMetrics(builder => builder
+            .AddAspNetCoreInstrumentation()
+            .AddOtlpExporter());
+}
+```
+
+#### List of metrics produced
+
+The instrumentation was implemented based on [metrics semantic
```
Comment (pellared) [inline src/OpenTelemetry.Instrumentation.AspNetCore/README.md:90]:
```suggestion
The instrumentation is implemented based on [metrics semantic
```

**Review by pellared — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.AspNetCore/README.md (hunk 1 above)
Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.AspNetCore/README.md]: Done 71301b6fd888981222fc2b3aa4d64116bacc7f08 together with the second occurence.

**Review by lachmatt — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.AspNetCore/README.md [hunk 2]
```diff
@@ -58,10 +63,43 @@ public void ConfigureServices(IServiceCollection services)
     services.AddOpenTelemetry()
         .WithTracing(builder => builder
             .AddAspNetCoreInstrumentation()
-            .AddJaegerExporter());
+            .AddOtlpExporter());
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Instrumentation.AspNetCore/README.md:66]: Any specific reason to use otlp here? If not, can we use console exporter? (Http one uses console)

Comment (vishweshbankwar) [reply src/OpenTelemetry.Instrumentation.AspNetCore/README.md:66]: missed the note on PR description. Could you please explain why?
`Console exporter from Http example is not good idea for the ASP.NET Core application`

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.AspNetCore/README.md]: Welll, my mind is leaving in the world where ASP.NET means IIS...

Fixed :- )

**Review by cijothomas — APPROVED** [review]

**Review by vishweshbankwar — APPROVED** [review]

**Pull request merged**